### PR TITLE
feat(icons): add creditcard svg to icons

### PR DESCRIPTION
**Pull request by dazziola**
This pull request adds a credit card SVG to the Paste Icons library.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- ✅  I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Comment by TheSisb** [comment]
Thanks for the contribution! I marked this PR as a `feat` (short for feature) since it adds an icon. We need to update our self-serve icon instructions, and I've made a ticket to do just that, but could you also run `yarn changeset` in the root of the paste repository? This will guide you to making a changelog entry, where you can simply write "Adding a creditcard icon" to the icons package.

File: .changeset/tender-cherries-tell.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@twilio-paste/icons': major
```
Comment (TheSisb) [inline .changeset/tender-cherries-tell.md:2]: Hmm, this should be a `minor`. Can you just tweak this real quick please?

Comment (dazziola) [reply .changeset/tender-cherries-tell.md]: Fixed that @TheSisb - didn't realise that the CLI accepted an empty selection for the prompt

Comment (TheSisb) [reply .changeset/tender-cherries-tell.md]: Yeah it's a little confusing. Thanks!

**Review by TheSisb — APPROVED** [review]

**Pull request merged**